pi-manage

```diff
@@ -837,32 +855,35 @@ def list(verbose):
     resolver_list = get_resolver_list()
 
     if not verbose:
-        for name, resolver in resolver_list.iteritems():
+        for (name, resolver) in resolver_list.items():
             print("{0!s:16} - ({1!s})".format(name, resolver.get("type")))
     else:
-        for name, resolver in resolver_list.iteritems():
+        for (name, resolver) in resolver_list.items():
             print("{0!s:16} - ({1!s})".format(name, resolver.get("type")))
             print("."*32)
             data = resolver.get("data", {})
-            for k, v in data.iteritems():
+            for (k, v) in data.items():
                 if k.lower() in ["bindpw", "password"]:
                     v = "xxxxx"
                 print("{0!s:>24}: {1!r}".format(k, v))
             print("")
 
 
 @realm_manager.command
```

Comment:
I think the command is redundant because we add a ``list`` command manually below?

Reply:
fixed in b065a00f